massgov/mayflower
# DP-9569 Remove raw filter from _patterns/01-atoms/08-lists/ordered-list.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from _patterns/01-atoms/08-lists/ordered-list.twig.

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/ [feature/DP-9569_atoms-lists-ordered-list] grep -r '08-lists/ordered-list.twig' patternlab/styleguide/source/_patterns
patternlab/styleguide/source/_patterns/03-organisms/by-author/rich-text-description.json:      "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/03-organisms/by-author/rich-text.json:      "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/binder-page-internal.json:    "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/board-decision.json:        "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/howto.json:            "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/howto.json:            "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/information-details.json:          "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/information-details.json:          "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/information-details~preContent-media.json:          "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/information-details~preContent-media.json:          "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/information-details~table-of-contents.json:          "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/regulation.json:        "path": "@atoms/08-lists/ordered-list.twig",
patternlab/styleguide/source/_patterns/05-pages/regulation.json:        "path": "@atoms/08-lists/ordered-list.twig",
 ~/MassIT/mayflower/ [feature/DP-9569_atoms-lists-ordered-list]
```

## Related Issue / Ticket

- [DP-9569](https://jira.mass.gov/browse/DP-9569)


## Steps to Test


1. Go to the following pages and check for any display issue with ordered lists, especially when they are nested.

- Binder
- Decision
- How to
- Information details
- Regulations

## Screenshots
Template nesting sample:
<img width="625" alt="pattern_lab_-_atoms-ordered-list" src="https://user-images.githubusercontent.com/9633303/43525287-5cb7a9b2-956f-11e8-9950-e667811e7b2c.png">



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
